src/protocol/sort/compose.rs

```diff
@@ -86,9 +86,9 @@ mod tests {
             let [ctx0, ctx1, ctx2] = make_contexts(&world);
 
             let sigma_and_randoms: [_; 3] = try_join_all([
-                shuffle_and_reveal_permutation(ctx0.narrow("shuffle_reveal"), sigma.len(), sigma0),
-                shuffle_and_reveal_permutation(ctx1.narrow("shuffle_reveal"), sigma.len(), sigma1),
-                shuffle_and_reveal_permutation(ctx2.narrow("shuffle_reveal"), sigma.len(), sigma2),
+                shuffle_and_reveal_permutation(ctx0.narrow("shuffle_reveal"), BATCHSIZE, sigma0),
+                shuffle_and_reveal_permutation(ctx1.narrow("shuffle_reveal"), BATCHSIZE, sigma1),
+                shuffle_and_reveal_permutation(ctx2.narrow("shuffle_reveal"), BATCHSIZE, sigma2),
             ])
```

Comment:
It might be good to avoid unrelated changes like this in future, but I agree that this is iffy.

Reply:
yea apologies for that - should've at least mentioned this in the description.